lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-docker/example-config/log4j.properties

```diff
@@ -0,0 +1,6 @@
+log4j.rootLogger=INFO, STDOUT
+log4j.appender.STDOUT=org.apache.log4j.ConsoleAppender
+log4j.appender.STDOUT.layout=org.apache.log4j.PatternLayout
+log4j.appender.STDOUT.layout.ConversionPattern=%5p [%t] (%F:%L) - %m%n
+
+log4j.logger.io.netty.handler.logging.LoggingHandler=WARN
```

Comment:
This line should not be needed. Netconf should no longer log debug/trace logs from netconf connection to INFO level.